doc/USAGE.md

```diff
@@ -33,6 +33,7 @@ Some key parameters you may want to start with are:
 * `memetic_<property>`: All the properties that only kick in if you use the `global` solver. The key one is `memetic_num_threads`, as we have enabled the evolutionary algorithm to solve on multiple threads.
 * `cost_threshold`: This solver works by setting up cost functions based on how far away your pose is, how much your joints move relative to the initial guess, and custom cost functions you can add. Optimization succeeds only if the cost is less than `cost_threshold`. Note that if you're adding custom cost functions, you may want to set this threshold fairly high and rely on `twist_threshold` to be your deciding factor whereas this is more of a guideline.
 * `twist_threshold`: Optimization succeeds only if the pose difference is less than `twist_threshold`. So, e.g., assuming your orientation was perfectly met, a `twist_threshold` of 0.001 would mean a 1 mm accuracy.
+* `approximate_solution_twist_threshold`: When using approximate IK solutions for applications such as endpoint servoing, `pick_ik` may sometimes return solutions that are significantly far from the goal frame. To prevent issues with such jumps in solutions, this parameter defines a maximum displacement. We recommend setting this to values around 0.01 - 0.05 (1-5 cm) for most applications.
```

Comment:
I don't see why "twist" needs to be in the name. Suggested renaming below. Obv it would be nice to have an angular threshold too but it doesn't have to be in this PR.

Another option would be something like `approximate_solution_allowable_joint_displacement`. If the change in angles would be too large -> reject it.

```suggestion
* `approximate_solution_distance_threshold`: When using approximate IK solutions for applications such as endpoint servoing, `pick_ik` may sometimes return solutions that are significantly far from the goal frame. To prevent issues with such jumps in solutions, this parameter defines a maximum displacement. We recommend setting this to values around 0.01 - 0.05 (1-5 cm) for most applications.
```

Reply:
Well. it's actually doing the calculation using the Eigen [`isApprox()`](https://eigen.tuxfamily.org/dox/classEigen_1_1DenseBase.html#ae8443357b808cd393be1b51974213f9c) function which actually does a matrix norm for the entire transform.

So this is only a distance comparison if you're doing position-only IK

Reply:
I think the plan is going to be merge this as is, and I'll put up a follow-on PR that replaces the twist with separate position and orientation thresholds.

As Andy pointed out, more parameters but also easier to understand.

Reply:
So do you want me to merge this and you'll make a separate PR or you want me to wait for the separate PR?

Reply:
Merging it now!